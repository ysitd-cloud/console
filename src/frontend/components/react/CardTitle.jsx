import React from 'react';
import classNames from 'classnames';
import PropTypes from 'prop-types';

export default function CardTitle(props) {
  const { className, children, ...other } = props
  const classes = classNames(className, 'card__title');
  return <div className={classes} {...other}>{children}</div>;
}

CardTitle.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node,
};

CardTitle.defaultProps = {
  className: '',
  children: null,
};
