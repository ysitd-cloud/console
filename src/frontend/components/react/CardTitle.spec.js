import React from 'react';
import renderer from 'react-test-renderer';
import CardTitle from './CardTitle';

/* globals describe, expect, it */

describe('React CardTitle Component', () => {
  it('match snapshot', () => {
    const component = renderer.create(<CardTitle className="xs12">foo</CardTitle>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
