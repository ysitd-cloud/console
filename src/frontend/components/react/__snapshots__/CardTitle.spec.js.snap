// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React CardTitle Component match snapshot 1`] = `
<div
  className="xs12 card__title"
>
  foo
</div>
`;
